ethstats/ethstats.go

```diff
@@ -238,26 +249,33 @@ func (s *Service) loop() {
 			case <-headSub.Err():
 				break HandleLoop
 			case delegateSignMsg := <-istDelegateSignCh:
```

Comment:
one nice (and small) refactor is to treat istDelegateCh as a "router". IMO on line 206 where we subscribe, we could just launch a go routiner like:
```go
go func() {
  ch := make(chan istanbul.MessageWithPeerIDEvent, istDelegateSignChanSize)
	subscription := s.backend.SubscribeNewDelegateSignEvent(ch)
  defer subscription.Unsubscribe()
  
  for {
    case msg := <- ch:
      // route msg either to signCh or sendCh
    case err := <- subscription.Err() // Handle subscrition errors that we are currently not doing!
    case <- quit // need to verify this, but some signal to stop (using go context module can work too)
  }
}
```